css-grid/Overview.bs

```diff
@@ -1117,10 +1144,10 @@ Implied Minimum Size of Grid Items</h3>
 		In particular, if grid layout is being used for a major content area of a document,
 		it is better to set an explicit font-relative minimum width such as ''min-width: 12em''.
 		A content-based minimum width could result in a large table or large image
-		stretching the size of the entire content area into an overflow zone,
```

Comment:
I think we probably do want to call out the potential for overflow here. The example is an item containing a very large picture (3x width of the viewport) and also lots of text. If the item stretches, that text will wrap to the width of the image, not of the viewport, and that's really awkward to read.